src/pybind/mgr/dashboard_v2/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -64,12 +64,11 @@
                  [loadingIndicator]="loadingIndicator"
                  [rowIdentity]="rowIdentity()"
                  [rowHeight]="'auto'">
-    <!-- Row Detail Template -->
-    <ngx-datatable-row-detail (toggle)="updateDetailView()">
```

Comment:
This should be kept this, because without it we can't show the details in the same row as the selected item. Which wasn't possible with the current solution, but may be now it's possible?

Reply:
I think the current implementation of details should be kept outside the tables, since it can get a bit long.

In the future if we find a better way to display the information we can just add this line again.